src/openrct2-ui/audio/AudioMixer.cpp

```diff
@@ -27,10 +27,10 @@ void AudioMixer::Init(const char* device)
     Close();
 
     SDL_AudioSpec want = {};
-    want.freq = 22050;
+    want.freq = 44100;
     want.format = AUDIO_S16SYS;
     want.channels = 2;
-    want.samples = 2048;
+    want.samples = 2048 * 8;
```

Comment:
I would be okay with 4096, I also experimented a bit with this and there was no difference other than memory consumption but that seems a bit too high.